.asf.yaml

```diff
@@ -4,6 +4,24 @@ notifications:
   issues: issues@trafficserver.apache.org
   pullrequests: github@trafficserver.apache.org
 github:
+  description: "Apache Traffic Server™ is a fast, scalable and extensible HTTP/1.1 and HTTP/2.0 compliant caching proxy server."
+  homepage: https://trafficserver.apache.org/
+  lables:
```

Comment:
labels ?

Reply:
Yes, it is part of the documentation from ASF.  I don't know how they are used.

https://cwiki.apache.org/confluence/display/INFRA/git+-+.asf.yaml+features

Reply:
i mean... isn't it a typos? you have "lables" instead.